keyboards/wheatfield/blocked65/keymaps/default/keymap.c

```diff
@@ -0,0 +1,37 @@
+#include QMK_KEYBOARD_H
+
+enum layers {
+  _QWERTY,
+  _RAISE,
+  _LOWER
+};
+
+enum keycodes {
+  QWERTY = SAFE_RANGE,
+  RAISE,
+};
```

Comment:
It doesn't look like these keycodes are used in any of the keymaps, so you can just remove the entire enum.

Reply:
Fixed